docs/product-migration-guide/src/main/asciidoc/migration-java-dependencies-proc.adoc

```diff
@@ -26,8 +26,21 @@ You must download the Maven repository for {PRODUCT} {PRODUCT_VERSION} and make
   <type>pom</type>
 </dependency>
 ----
+endif::DM[]
+ifdef::PAM[]
+[source,xml]
+----
+<dependency>
+  <groupId>org.jboss.bom.rhba</groupId>
```

Comment:
rhpam? Seach for rhba.

Reply:
Shoot. It's because that got re-added per SME feedback after my search/replace efforts and slipped through the cracks. Thank you!